src/main/java/org/jabref/gui/search/GlobalSearchBar.java

```diff
@@ -143,7 +143,7 @@ public void actionPerformed(ActionEvent e) {
                     updateOpenCurrentResultsTooltip(globalSearch.isSelected());
                     focus();
                     event.consume();
-                } else if (keyBinding.get().equals(KeyBinding.CLEAR_SEARCH)) {
+                } else if (keyBinding.get().equals(KeyBinding.CLOSE)) {
```

Comment:
I would prefer a switch case here like in the other dialogs to have it consistent

Reply:
Me too, but I'm too lazy right now to change it 😄